content/_components/shopify-admin-v2/index.md

```diff
@@ -65,37 +65,38 @@ Retrieve all the updated or created objects within a given time range.
 
 * **Object Type** - (dropdown, required): Object-type to lookup on. E.g `Customers`.
 
-* **Timestamp field to poll on** - (dropdown, required): Can be either Last Modified or Created dates (updated or new objects, respectively)
+* **Timestamp field to poll on** - (dropdown, required): Can be either `Last Modified` or `Created dates` (updated or new objects, respectively).
 
-* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost
-* **You can provide additional fields here** - (string, optional): Resulting object can be expanded using GraphQL request, this field represent content from each `edges.node`, it may affect on query cost
+* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost.
 
-<details close markdown="block"><summary><strong>Example for customer</strong></summary>
-
-  ```Graphql
-    metafields(first: 2) {
-      edges {
-        node {
-          namespace
-          key
-          value
-        }
-      }
-    }
-    addresses {
-      address1
-      country
-    }
-  ```
-
-</details>
+* **You can provide additional fields here** - (string, optional): Resulting object can be expanded using GraphQL request, this field represent content from each `edges.node`, it may affect on query cost.
```

Comment:
* **You can provide additional fields here** - (string, optional): The resulting object can be expanded using GraphQL request, this field represents content from each `edges.node`, it may affect query cost.